dogs.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ADOPTADOG</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+  </head>
+
+  <body>
+      <header>
+        <div class='float-right'>
+          <img src="images/cart-icon.svg" alt="Shopping cart icon." width="30">
+          $1234.56<br>
+          <button>Checkout</button>
+        </div>
+        <img src="images/adoptadog-logo.svg" alt="Profile of a brown dog, staring at something." width="175">
+        <h1>ADOPTADOG</h1>
+        <h3>Seattle's premier dog adoption agency.</h3>
+      </header>
+
+      <nav>
+        <ul>
+          <li><a href="index.html" target="_blank">Home</a>|</li>
+          <li><a href="dogs.html" target="_blank">Dogs</a>|</li>
+          <li><a href="blog.html" target="_blank">Blog</a>|</li>
+        </ul>
+      </nav>
+
+      <section class="flexbox-container">
+        <h3>Here are all of our Dogs!</h3>
```

Comment:
this h3 element might not be helping with your layout issues.

Reply:
Remember, when you use display flex on a container, all of it's direct children get the flex box treatment. In your case the h3 as well as the articles are getting it.